### PR TITLE
Add landing page services

### PR DESCRIPTION
**Pull request by filipebfernandes**

### CODE REVIEW

File: src/components/Service/index.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  @include Breakpoint-mobileOnly {
+    margin-bottom: $spacing-56;
+  }
+}
+
+.services:not(:last-child) {
+  margin-left: $spacing-12;
+
+  @include Breakpoint-mobileOnly {
+    margin-left: 8px;
```
Comment (taniaprocha) [inline src/components/Service/index.css]: Why `8px`? We don't have this spacing

Comment (jfranciscosousa) [reply src/components/Service/index.css]: 💀

File: src/components/Service/index.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  @include Breakpoint-mobileOnly {
+    width: 8px;
+  }
+}
+
+.type {
+  position: absolute;
+
+  display: inline;
+  height: fit-content;
```
Comment (jfranciscosousa) [inline src/components/Service/index.css]: Don't use this, it's unsupported by anything not named Google Chrome

File: src/components/Service/index.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  margin-top: 40px;
```
Comment (jfranciscosousa) [inline src/components/Service/index.css]: Forget margin, use `top: calc($asterisk-size-mobile + $spacing-16)` and `left: $spacing-14`

File: src/components/Service/index.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+  transform: rotate(90deg) translateY(-70%);
```
Comment (jfranciscosousa) [inline src/components/Service/index.css]: With the top and left attribute you just need to translate 25% of the size of the text, so it aligns with the asterisk.

**Comment by filipebfernandes** [comment]
@jfranciscosousa made the changes, now it's using only design spacings.

File: src/components/Service/index.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    left: calc($line-height-h3-mobile / 2);
+  }
+}
+
+.service-description {
+  display: flex;
+  flex-direction: column;
+}
+
+.image {
```
Comment (jfranciscosousa) [inline src/components/Service/index.css]: ‼️

Comment (filipebfernandes) [reply src/components/Service/index.css]: 🇷🇼

File: src/components/Service/index.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
+@import "../../styles/layout.css";
+@import "../../styles/breakpoints.css";
+@import "../../styles/variables.css";
```
Comment (taniaprocha) [inline src/components/Service/index.css]: Rebase and change to `@import "../../styles/all.css"`

File: src/sections/HomeServices/index.css [hunk 7]
```diff
@@ -0,0 +1,35 @@
+@import "../../styles/layout.css";
+@import "../../styles/breakpoints.css";
```
[the remark this answers is not in the document]
Comment (filipebfernandes) [reply src/sections/HomeServices/index.css]: 👍

File: src/sections/HomeServices/index.css [hunk 8]
```diff
@@ -0,0 +1,34 @@
+@import "../../styles/all.css";
+
+.title {
+  @include Breakpoint-tabletAndAbove {
+    max-width: $four-columns;
+  }
```
Comment (jfranciscosousa) [inline src/sections/HomeServices/index.css]: Use newlines

File: src/sections/HomeServices/index.css [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  @include Breakpoint-mobileOnly {
+    margin-right: auto;
+  }
+  margin-bottom: $spacing-56;
```
Comment (jfranciscosousa) [inline src/sections/HomeServices/index.css]: Also here, always put newlines after mixin usage.

File: src/sections/HomeServices/index.css (hunk 8 above)
Comment (jfranciscosousa) [reply src/sections/HomeServices/index.css]: Only after

**Pull request merged**